packages/block-library/src/image/image.js

```diff
@@ -473,6 +476,7 @@ export default function Image( {
 						width: parseInt( currentWidth + delta.width, 10 ),
 						height: parseInt( currentHeight + delta.height, 10 ),
```

Comment:
do we know why setting the attributes here is _not_ considered a persistent change already?

Reply:
It would be great to expand the functionality of the `transientEdit` property. Rather than only checking for the whole object, we could have paths, for example `blocks.attributes.width`. Though we don't want to write out every attribute like this, so maybe we could have an array for excludes as well. Example:

**block.json**
```json
persistentEdits: ["blocks.attributes.width"]
```

or we could do this at attribute level: (either `transient: false` or `persisent: true`)
**block.json**
```json
"attributes": {
    "width": {
        "type": "number",
        "persistent": true,
    },
    "height": {
        "type": "number",
        "persistent": true,
    },
}
```

Reply:
It seems like we had this discussion several times now. I'm unsure about the latest decisions though. cc @ellatrix It seems like we didn't settle on a path forward yet.

Reply:
Maybe we should revisit the logic in the reducer for updating the same attribute to not create an undo level? Wether or not this is needed seems to depend on the user interaction rather than the data updating